test/riscv-tests/CMakeLists.txt

```diff
@@ -3,19 +3,19 @@
 # On success or failure they write to the `tohost` symbol. See this code:
 # https://github.com/riscv/riscv-test-env/blob/4fabfb4e0d3eacc1dc791da70e342e4b68ea7e46/p/riscv_test.h#L200
 
-file(GLOB elfs_rv32d CONFIGURE_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/rv32*.elf")
-file(GLOB elfs_rv64d CONFIGURE_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/rv64*.elf")
+# file(GLOB elfs_rv32d CONFIGURE_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/rv32*.elf")
```

Comment:
So I guess here we would need to edit the json?

Maybe the easiest way would be to make it a template and use configure_file() to replace `@XLEN_LOG2@`, etc. to generate a config_rv32d.json, config_rv64d.json in the build dir?